server/simple_server.go

```diff
@@ -112,6 +112,8 @@ func (s *SimpleServer) Start() error {
 	srv := http.Server{
 		Handler:        RegisterAccessLogger(s.cfg, s),
 		MaxHeaderBytes: maxHeaderBytes,
+		ReadTimeout:    readTimeout,
+		WriteTimeout:   writeTimeout,
```

Comment:
I agree, RPC needs some love and refactoring, but it's a bit out of the scope of this change. (Maybe we make an issue/enhancement?)

In the end, I imagine/hope the guts of the RPC server would get replaced by a bunch of go-kit transports and middleware, which is why I haven't coupled the two servers thus far.